docs/source/search-runs.rst

```diff
@@ -156,6 +156,23 @@ You can search using the following run attributes contained in :py:class:`mlflow
   attributes.run_id = 'a1b2c3d4'
   attributes.run_id IN ('a1b2c3d4', 'e5f6g7h8')
 
+Datasets
+~~~~~~~~~~~~~~
+
+You can search using the following dataset attributes contained in :py:class:`mlflow.entities.Dataset`: ``name``, ``digest``, ``source_type``, ``source``, ``schema``, and ``profile``. 
```

Comment:
I think we should start with `name`, `digest`, and `context`. It's unlikely / unclear to me that users would have an entire source / schema / profile dictionary to search against, and I also don't think it's likely that users will need to search based on source_type (can add it later, if necessary).

This PR only seems to test `name`, `digest`, and `context`, so removing these other attributes will save us time :)

Reply:
agreed. the story for searching those other fields doesn't make too much sense for now.